Library/Formula/iojs.rb

```diff
@@ -1,4 +1,8 @@
+require "language/javascript"
+
 class Iojs < Formula
+  include Language::JS
```

Comment:
I remember we talked about exposing `Formula.system`'s behaviour elsewhere? Did we ever do that? If not: maybe this is a good time to do so?